recipes/pyaes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyaes" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 02c1b1405c38d3c370b085fb952dd8bea3fadcee6411ad99f312cc129c536d8f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pyaes
+
+about:
+  home: "https://github.com/ricmoo/pyaes"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE.txt"
+  summary: "Pure-Python Implementation of the AES block-cipher and common modes of operation"
+  doc_url: "https://pypi.org/project/pyaes/"
+  dev_url: "https://github.com/ricmoo/pyaes"
+
+extra:
+  recipe-maintainers:
+    - mwilson8
+    - oblute
+    - benhuff
```

Comment:
@oblute @benhuff - are you willing to be maintainers?

Reply:
yes!

Reply:
yes